prog/mutation.go

```diff
@@ -374,6 +374,29 @@ func (t *BufferType) mutate(r *randGen, s *state, arg Arg, ctx ArgCtx) (calls []
 	case BufferText:
 		data := append([]byte{}, a.Data()...)
 		a.data = r.mutateText(t.Text, data)
+	case BufferCompressed:
+		data := a.Data()
+		if len(data) == 0 {
+			return
+		}
+		data, err := Decompress(data)
+		if err != nil {
+			panic(fmt.Sprintf("could not decompress data: %v", err))
+		}
+		if len(data) == 0 {
+			return // Do not mutate empty data.
+		}
+		hm := MakeGenericHeatmap(data)
+		// At least one mutation, up to about one mutation every 256 KB
+		numMutations := r.Intn(len(data)/(1<<18)+1) + 1
```

Comment:
I think we need to base this on the number of non-trivial bytes in the image (hm.length) rather than full uncompressed image size.
This may be a good method on the heat map.